### ISSUE TITLE
Steering onboarding - @mrbobbytables, @palnabarun, @BenTheElder

### ISSUE THREAD

**Issue by justaugustus** [kubernetes/steering#256]
# Steering Onboarding

This document covers steps needed to onboard new Steering Committee
members and off-board emeritus members.

_NOTE(@justaugustus): These items were initially copied from https://github.com/kubernetes/steering/issues/219. I'll be reviewing and cleaning up this list as we action on the items._

## Initial steps

 - [x] All new members must complete the [Inclusive Leadership Training]
within 30 days from the date of their appointment.
   - [ ] @mrbobbytables certificate ID number: [pending](https://github.com/kubernetes/steering/issues/256#issuecomment-1267064363)
   - [x] @palnabarun certificate ID number: [LF-w8xm0436k7](https://user-images.githubusercontent.com/10010419/194336081-097fb748-90e5-448f-a700-635b61396fdf.png)
   - [x] @BenTheElder certificate ID number: [LF-opnkrmypm4](https://github.com/kubernetes/steering/files/9701542/benjamin-elder-b657cdeb-95e7-4deb-8e22-a75da6cf8954-certificate.pdf)
 - [x] Update the following files in the respective repos:
   - [x] [kubernetes/steering]: `OWNERS_ALIASES` and `README.md`: https://github.com/kubernetes/steering/pull/257
   - [x] [kubernetes/community]:
     - [x] `sigs.yaml` (SC members and liaisons) and auto-generated content: https://github.com/kubernetes/community/pull/6905
     - [x] slack usergroup - `communication/slack-config/usergroups.yaml`, `communication/slack-config/users.yaml`: https://github.com/kubernetes/community/pull/6905
   - [x] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`: https://github.com/kubernetes/org/pull/3746
   - [x] [kubernetes/website]: `OWNERS_ALIASES`: https://github.com/kubernetes/website/pull/37170
   - [x] [kubernetes/funding]: `OWNERS_ALIASES`:  https://github.com/kubernetes/funding/pull/34
   - [x] [kubernetes/k8s.io]: https://github.com/kubernetes/k8s.io/pull/4300
     - [x] update steering@kubernetes.io and steering-private@kubernetes.io in `groups/groups.yaml`: 
     - [x] `OWNERS_ALIASES`: 
 - [x] Add new members to the `steering-private` slack channel
 - [x] Add new members to the CNCF Service Desk and remove emeritus members
 - [x] Add new members to the [cncf-kubernetes-maintainers] mailing list and remove emeritus members
 - [x] Add new members to the [Public CNCF Maintainer List] and remove emeritus members: https://github.com/cncf/foundation/pull/430
 - [x] Transfer Community Group [liaison assignments] from emeritus members to new members: https://github.com/kubernetes/community/pull/6907
 - [x] If a Google Workspace [top-level account] is owned by an emeritus member,
  transfer it to an existing member (currently owned by @dims, @parispittman, and @mrbobbytables)
   - [x] @dims to handoff to @justaugustus
 - [x] (N/A) If an emeritus member is one of the @kubernetesio Twitter account contributors,
  transfer access to an existing member
  - Currently owned by @parispittman and @mrbobbytables:  https://github.com/kubernetes/community/blob/35d8e8cc6a5165352e92ec17d5bf9e892e653bc7/communication/marketing-team/social-guidelines.md
 - [x] (N/A) If [Zoom license credentials] were owned by a majority of emeritus
  members, share them with existing members. The Zoom host key is
  accessible to all members and is a pinned message in the `steering-private`
  slack channel
  - Currently owned by @parispittman, @cblecker, and @mrbobbytables: https://github.com/kubernetes/steering/blob/5a0edcffc51e819a8349fee34f32b6568d2e4323/README.md#zoom-admins
 - [x] Add new members to 1password, remove emeritus members: @mrbobbytables handled this
 - [ ] Set up a private meeting to go over backlog

### After handoff meeting

 - [x] Remove emeritus members from the `steering-private` slack channel

---

I'll take point on the PRs:
/assign @justaugustus
cc: @mrbobbytables @palnabarun @BenTheElder

[Inclusive Leadership Training]: https://github.com/kubernetes/steering/blob/main/charter.md#inclusive-leadership-training
[kubernetes/steering]: https://github.com/kubernetes/steering
[kubernetes/community]: https://github.com/kubernetes/community
[kubernetes/org]: https://github.com/kubernetes/org
[kubernetes/website]: https://github.com/kubernetes/website
[kubernetes/funding]: https://github.com/kubernetes/funding
[kubernetes/k8s.io]: https://github.com/kubernetes/k8s.io
[liaison assignments]: https://git.k8s.io/community/liaisons.md
[cncf-kubernetes-maintainers]: https://lists.cncf.io/g/cncf-kubernetes-maintainers
[Public CNCF Maintainer List]: https://github.com/cncf/foundation/blob/master/project-maintainers.csv
[top-level account]: https://github.com/kubernetes/steering/README.md#top-level-accounts
[Zoom license credentials]: https://github.com/kubernetes/steering/README.md#zoom-admins

**Comment by dims** [comment]
@palnabarun @BenTheElder please off-board me when you are cutting those PR(s) :) thanks!

**Comment by justaugustus** [comment]
/assign @BenTheElder @palnabarun @mrbobbytables 

Ben/Nabarun -- I've opened https://github.com/kubernetes/steering/pull/257, https://github.com/kubernetes/community/pull/6905, and I'm handing off the K8s Community items to you.

Let me know if you have any questions.

I'll take care of the CNCF-level tasks!

**Comment by BenTheElder** [comment]
>  @BenTheElder certificate ID number:

The last time I took this was becoming SIG Testing Chair I think (!), re-certified today:
[benjamin-elder-b657cdeb-95e7-4deb-8e22-a75da6cf8954-certificate.pdf](https://github.com/kubernetes/steering/files/9701542/benjamin-elder-b657cdeb-95e7-4deb-8e22-a75da6cf8954-certificate.pdf)


ID: `LF-opnkrmypm4`

**Comment by mrbobbytables** [comment]
I recertified but I haven't gotten a pdf / id copy for some reason. Here is a screenshot of the completion screen though
![Screen Shot 2022-10-04 at 9 05 27 AM](https://user-images.githubusercontent.com/6500863/193841101-7170f012-e587-4e9c-8638-55f49ddb9720.png)

**Comment by BenTheElder** [comment]
> [kubernetes/website](https://github.com/kubernetes/website): OWNERS_ALIASES:

https://github.com/kubernetes/website/pull/37170

**Comment by palnabarun** [comment]
Completed the training. ID: `LF-w8xm0436k7`

![image](https://user-images.githubusercontent.com/10010419/194336081-097fb748-90e5-448f-a700-635b61396fdf.png)

**Comment by BenTheElder** [comment]
CNCFSD-1450: service ticket tracking remaining updates.

I met with @mrbobbytables to try to figure out getting his certificate and the training site seems to be broken for him, I watched him complete the course and then the entire "certificates of completion" tab doesn't render at all on his account in multiple web browsers.

**Comment by BenTheElder** [comment]
>  Add new members to the CNCF Service Desk and remove emeritus members

I missed this invite and need to fix this, but otherwise I think this issue is about done

**Comment by BenTheElder** [comment]
I have access to the service desk, bob's account seems to be broken still so we can't get the certificate. But I can assert that he took it (we jumped on a call to try to debug accessing the certificate).

I think we can close this, and do a follow-up re: backlog sometime involving @cpanato as well re: https://github.com/kubernetes/steering/issues/260, in the meantime @palnabarun and I have attempted to get context on the backlog via other means.
